code/src/components/CompletedList.js

```diff
@@ -0,0 +1,16 @@
+import React from "react"
+import { useSelector } from "react-redux"
+import { CompletedItem } from "components/CompletedItem"
+
+export const CompletedList = () => {
+  const todoItems = useSelector(state => state.todoList.completedItems)
+
+  return (
+    <ul>
+      {todoItems.length > 0 && <h2>Completed todo</h2>}
+      {todoItems.map(item => (
+        <CompletedItem key={item.id} item={item} />
```

Comment:
This component is exactly the same as the ` <UncompletedItem />` right?
In that case, its better that you create one called **TodoItem** or similar that you can use in both lists.